packages/xo-web/src/common/intl/messages.js

```diff
@@ -1523,8 +1528,7 @@ const messages = {
   ha: 'HA',
   srHaTooltip: 'SR used for High Availability',
   nestedVirt: 'Nested virtualization',
-  nestedVirtualizationWarning:
-    'Unstable feature, insecure for the host, usage is discouraged. Click for more details.',
+  nestedVirtualizationWarning: 'Unstable feature, insecure for the host, usage is discouraged. Click for more details.',
```

Comment:
Prettier ?

Reply:
Yes.

Reply:
It's because the file was incorrectly formatted in a previous PR.

I'll improve the CI to detect this.